java/ql/test/query-tests/Telemetry/AutomodelFrameworkModeExtractCandidates/java/nio/file/Files.java

```diff
@@ -0,0 +1,15 @@
+package java.nio.file;
```

Comment:
There is a `stubs` directory generally used for such stub library definitions. Could this be placed there? I see no existing examples of stubs for JDK classes so worth checking with the Java team how they usually handle these - it may be that we assume the JDK is already a dependency and so is extracted.

Reply:
after a bit of back and forth, I got this to work :) [2a00bec](https://github.com/github/codeql/pull/13788/commits/2a00becc3c29de3d447f1d6fabdd24d837bd9a85)

Reply:
@atorralba or @jcogs33 - could one of you confirm that we're using the stubs directory correctly? Or would you prefer we revert commit [2a00bec](https://github.com/github/codeql/pull/13788/commits/2a00becc3c29de3d447f1d6fabdd24d837bd9a85)?

Reply:
JDK stubs shouldn't be needed normally. I _think_ what you're doing is similar to the test case I created on [this PR](https://github.com/github/codeql/pull/12680#issuecomment-1496665864). :smile: 
IMO, these stubs probably should not live in the `stubs` directory since they're essentially being used as source code, and they're specific to one test case instead of being generic stubs (but I'll defer to @atorralba's opinion on placement).

Reply:
I definitely think these shouldn't be in the `stubs` directory to not add confusion about the necessity of JDK stubs. In this case, since these are a few calls only, I'd prefer a source file in the test directory (say, `A.java`) that makes the specific calls you want to test the methods of. That way you ensure that they're extracted, and then your CodeQL code should be able to find the method definitions.

Reply:
OK, I sort of expected this answer :) I have removed the last commit that moved those tests to the stubs directory. Resolving this conversation :)